{"author":"dotnet-bot","ms.author":"dotnetcontent","manager":"wpickett","breadcrumb_path":"/dotnet/toc.json","ms.topic":"managed-reference","ms.prod":".net","search.ms_sitename":"Docs","search.ms_docsetname":"docs-internal","version":0,"locale":"en-us","site_name":"Docs","search.ms_product":"MSDN","depot_name":"MSDN.docs-internal","original_ref_skeleton_git_url":"https://github.com/dotnet/docs-internal/blob/master/api/System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.yml","open_to_public_contributors":false,"api_name":["System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord..ctor","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.Clone","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.FaultSource","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.ToString","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.UnhandledException"],"api_location":["System.Activities.dll"],"topic_type":["apiref"],"api_type":["Assembly"],"f1_keywords":["System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord","System::Activities::Tracking::WorkflowInstanceUnhandledExceptionRecord","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.#ctor","System::Activities::Tracking::WorkflowInstanceUnhandledExceptionRecord::#ctor","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.Clone","System::Activities::Tracking::WorkflowInstanceUnhandledExceptionRecord::Clone","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.FaultSource","System::Activities::Tracking::WorkflowInstanceUnhandledExceptionRecord::FaultSource","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.ToString","System::Activities::Tracking::WorkflowInstanceUnhandledExceptionRecord::ToString","System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord.UnhandledException","System::Activities::Tracking::WorkflowInstanceUnhandledExceptionRecord::UnhandledException"],"dev_langs":["csharp"],"updated_at":"2017-03-24 01:41 AM","document_id":"c41824ce-1111-c0c4-21b2-cc596433c01e","content_git_url":"https://github.com/dotnet/docs/blob/master/xml/System.Activities.Tracking/WorkflowInstanceUnhandledExceptionRecord.xml","layout":"Reference","pagetype":"Reference","title":"WorkflowInstanceUnhandledExceptionRecord class | Microsoft Docs","description":"Contains the data sent to a tracking participant by the run-time tracking infrastructure when a workflow instance encounters an unhandled exception.\n","toc_asset_id":"api/_splitted/System.Activities.Tracking/toc.json","toc_rel":"_splitted/System.Activities.Tracking/toc.json","source_url":"","ms.assetid":"System.Activities.Tracking.WorkflowInstanceUnhandledExceptionRecord","canonical_url":"https://ppe.docs.microsoft.com/en-us/dotnet/api/system.activities.tracking.workflowinstanceunhandledexceptionrecord","is_dynamic_rendering":true,"content_type":"application/json","publish_version":"{\"opbuild.templates.docs\":\"1.29.3\",\"commit_id\":\"b3a23883507f33cbc5269a49bae8b286e3a2a4b9\",\"branch\":\"master\",\"opbuild.scripts\":\"1.29.3\",\"opbuild.htmltopdftool\":\"1.29.3\",\"opbuild.gitcontributorinformationresolver\":\"1.29.3\",\"opbuild_service\":\"1.29.125.0\",\"docfx.console\":\"2.14.1\",\"opbuild.resolvedependencyconsole\":\"1.29.3\",\"opbuild.schemavalidationconsole\":\"1.29.3\",\"opbuild.redirectionresolver\":\"1.29.3\",\"opbuild.templates.intellisense\":\"1.29.3\",\"dependencies\":[{\"branch\":\"preview\",\"path\":\"_themes\",\"url\":\"https://github.com/Microsoft/templates.docs.msft\",\"commit_id\":\"29e90ce5b9d5929c589caa705c6513cef19b8b33\",\"opbuild.templates.common\":\"1.29.3\"},{\"branch\":\"master\",\"path\":\"_themes.pdf\",\"url\":\"https://github.com/Microsoft/templates.docs.msft.pdf\",\"commit_id\":\"b2158fbf2c62f32aff19e6f61397f4caa40b05f7\",\"opbuild.templates.common\":\"1.29.3\"}]}"}